bigtable/admin.go

```diff
@@ -119,6 +120,71 @@ func (ac *AdminClient) backupPath(cluster, backup string) string {
 	return fmt.Sprintf("projects/%s/instances/%s/clusters/%s/backups/%s", ac.project, ac.instance, cluster, backup)
 }
 
+// EncryptionInfo represents the encryption info of a table.
+type EncryptionInfo struct {
+	// TODO: compare to https://github.com/googleapis/java-bigtable/pull/656/files#diff-10655a12f2a8430533dca76435f6829a1a0bc978ada1295529b59a46afbcd95fR28
+	EncryptionStatus *status.Status // TODO: should this be wrapped as EncryptionType has been?
+	EncryptionType   EncryptionType
+	KMSKeyVersion    string
+}
+
+func newEncryptionInfo(pbInfo *btapb.EncryptionInfo) EncryptionInfo {
+	info := EncryptionInfo{}
+	info.EncryptionStatus = pbInfo.EncryptionStatus
+	// TODO: could also just return this as a string, but wrapped
+	info.EncryptionType = EncryptionType(pbInfo.EncryptionType.Number())
+	info.KMSKeyVersion = pbInfo.KmsKeyVersion
+
+	return info
+}
+
+type EncryptionType int32
+
+const (
+	// Encryption type was not specified, though data at rest remains encrypted.
```

Comment:
@tritone I made this iota as you recommended.

Curious though, since these need to match the pb, would it be better to
```
EncryptionTypeUnspecified EncryptionType = EncryptionType(btapb.EncryptionInfo_ENCRYPTION_TYPE_UNSPECIFIED)
```

these should be functionally equivalent.

Reply:
I think what you have now is fine. Maybe add a comment that you expect these to match the proto values exactly.